ebean-test/src/test/java/misc/migration/v1_0/ETable.java

```diff
@@ -0,0 +1,90 @@
+package misc.migration.v1_0;
+
+import java.util.List;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import io.ebean.annotation.DbComment;
+import io.ebean.annotation.History;
+import io.ebean.annotation.Index;
+
+@Table(name = "`table`")
+@Entity
+@History
+public class ETable {
```

Comment:
This is the "Reserved-keyword" entity. It uses a keyword for each fied and tries to address as many as possible features during DDL generation